apps/server/src/modules/tool/controller/api-test/tool.api.spec.ts

```diff
@@ -0,0 +1,307 @@
+import { EntityManager, MikroORM } from '@mikro-orm/core';
```

Comment:
The API tests should try all parameters available on the API, users with different roles, as well as relevant error cases. Are there any further scenarios with different parameters to test with? At least error cases should be added here. (not authenticated, no permission, bad request data)